charts/kong-collectorapi/values.yaml

```diff
@@ -0,0 +1,49 @@
+image:
+  # repository: kong-docker-kong-brain-immunity-base.bintray.io/kong-brain-immunity
+  # tag: latest
+  pullPolicy: IfNotPresent
+  repository: kong-docker-kong-brain-internal-builds.bintray.io/kong-brain
```

Comment:
This seems wrong. We should keep line 2 as that's the one we want users to use. Same applies for the tag.

Reply:
Just temporarily using internal master builds until I've made a release for collector and collector-plugin is in an enterprise build.